src/component/page/deltakerliste-veileder/IngenDeltakere.tsx

```diff
@@ -6,7 +6,7 @@ export const IngenDeltakere = (): JSX.Element => (
 	<div className={styles.ingenDeltakere}>
 		<GuidePanel poster>
         Du er logget inn som veileder i deltakeroversikten. Her kan du holde oversikt over deltakere som du følger opp.
-
+			<p />
```

Comment:
En random `p`? For linjebrekk inne i tekst kan man bruke `<br />` 🙂

Reply:
Takk :D